clippy_lints/src/return_self_not_must_use.rs

```diff
@@ -65,11 +87,13 @@ fn check_method(cx: &LateContext<'tcx>, decl: &'tcx FnDecl<'tcx>, fn_def: LocalD
         if !is_must_use_ty(cx, ret_ty);
 
         then {
-            span_lint(
+            span_lint_and_help(
                 cx,
                 RETURN_SELF_NOT_MUST_USE,
                 span,
                 "missing `#[must_use]` attribute on a method returning `Self`",
+                None,
+                "consider adding the `#[must_use]` attribute to the method or `Self` type"
```

Comment:
I would word it as is:

```suggestion
                "consider adding the `#[must_use]` attribute to the method or directly to the `Self` type"
```